### PR TITLE
Add a new optional "file" field for plugin list

### LINKED ISSUE #129: Allow `plugin list` to output plugins slugs
Hello,

It would be great to allow the command `plugin list` to allow `slug` as a value for the fields parameters.

Command: `wp plugin list --fields=slug`

Expected output:

```
+-------------------------------------------+
| slug                                      |
+-------------------------------------------+
| akismet.php                               |
| hello.php                                 |
| iwp-client/init.php                       |
| plugins-load-order/plugins-load-order.php |
+-------------------------------------------+
```

### PR DESCRIPTION
**Pull request by mustafauysal**
"file" parameter represents the plugin file

fixes #129

Props also to @vladutilie for another fix for the same issue in #185.

### CODE REVIEW

File: src/Plugin_Command.php [hunk 1]
```diff
@@ -653,7 +655,7 @@ protected function get_item_list() {
 			}
 
 			$duplicate_names[ $name ][] = $file;
-			$items[ $file ]             = [
+			$items[ $file ] = [
```
Comment (wojsmol) [inline src/Plugin_Command.php]:
```suggestion
	$items[ $file ]             = [
```

Comment (wojsmol) [inline src/Plugin_Command.php]: revert this change

Comment (mustafauysal) [reply src/Plugin_Command.php]: done 9693df372b720a7f6fb9686a92ba008ad9a80992
